view/components/main.html

```diff
@@ -1,38 +1,29 @@
 <div data-ng-init="initStack()">
-    <span id="chat-screen">
-        -- Chat Screen --
+    <span id="chat-screen" style="color:black;">
+         Chat Screen 
         <hr>
     </span>
     <!-- message field -->
     <div id="message-bar">
         <div id="message-input-parent">
-            <input type="text" name="" id="message-input" data-ng-model="message" data-ng-change="removeMessage()">
-            <button id="message-bar-send" data-ng-click="addMessagesToStack()">
+            <input type="text"  name="" id="message-input" data-ng-model="message" data-ng-change="removeMessage()">
+            <button id="message-bar-send"  data-ng-click="addMessagesToStack()" nowrap>
                 Send
             </button>
             <!-- banner -->
             <div id="banner-listen" data-ng-show="showLoaderListening">
                 <span>Listening ... </span>
-                <img src="../assest/icons/load.gif" alt="listening to you. Please keep speaking.." id="loader" srcset="">
+                <img src="../assets/icons/load.gif" alt="listening to you. Please keep speaking.." id="loader" srcset="">
             </div>
-            <img src="../assest/icons/mic.png" alt="Click to Start/Stop speaking" id="button" style="height: 40px;margin-left: 15px;" data-ng-click="toggleStartStop()" />
+            <img src="../assets/icons/mic.png" alt="Click to Start/Stop speaking" id="button" style="height: 40px;margin-left: 15px;" data-ng-click="toggleStartStop()" />
         </div>
     </div>
 
     <!-- chat screen -->
 
-    <div data-ng-show="messageStack.length !== 0" id="stackArea-parent">
-        <div data-ng-repeat="message in messageStack" id="stackArea">
-            <div data-ng-if="message.sender === 'you'" class="chat-screen-messages-you">
-                {{message.message}}
-                <span>
-                    <sub class="show-user-creds">
-                        {{message.sender}}
-                        {{message.time}}
-                    </sub>
-                </span>
-            </div>
```

Comment:
Why have u removed the Bot code? just copy that part from the original repo. Rest all mergeable!
Please make sure that u never remove the original code.